### PR TITLE
Use long class names in html output files

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### Reference Issues/PRs
#364 
#### What does this implement/fix? Explain your changes.
new boolean parameter named `use_long_names` added to `save_html` function which provides us to use long class names in html output files

### CODE REVIEW

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@sadrasabouri Thanks for your effort 💯 
I suggest `shortener` (default : `True`) instead of `use_long_names`, whats your idea?

File: pycm/pycm_obj.py [hunk 1]
```diff
@@ -315,7 +315,7 @@ def save_html(
         :type summary : bool
         :param alt_link: alternative link for document flag
         :type alt_link: bool
-        :param use_long_names: flag which shows if long names are used
+        :param shortener: flag which shows if long names are used
         :type use_long_name: bool
```
Comment (sepandhaghighi) [inline pycm/pycm_obj.py:319]: :type shortener ...

File: pycm/pycm_output.py [hunk 2]
```diff
@@ -100,7 +100,7 @@ def html_table(
     :type rgb_color : tuple
     :param normalize : save normalize matrix flag
     :type normalize : bool
-    :param use_long_names: flag which shows if long names are used
+    :param shortener: flag which shows if long names are used
```
Comment (sepandhaghighi) [inline pycm/pycm_output.py:103]: :type shortener ...

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

**Review by sepandhaghighi — APPROVED** [review]
LGTM 💯 
@alirezazolanvari Please review this PR, if everything is okay, merge it after approval 🥇

**Review by alirezazolanvari — APPROVED** [review]
Nice work! LGTM

**Pull request merged**